concent_api/core/migrations/0008_store_message_timestamp_fix.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from common.helpers import deserialize_message
+from common.helpers import parse_timestamp_to_utc_datetime
+
+
+def switch_stored_message_timestamp_to_message_creation_time(apps, _schema_editor):
+    StoredMessage = apps.get_model('core', 'StoredMessage')
+
+    for stored_message in StoredMessage.objects.all():
+        stored_message.timestamp = parse_timestamp_to_utc_datetime(
+            deserialize_message(stored_message.data).timestamp
```

Comment:
Shouldn't it be `.tobytes()` ?

Reply:
I've change it, but it looks like it didn't make any different. 
P.S.
One other think about this migration is that all existing messages `created_at` field is going to get change to "current" timestamp. I hope it's not a problem?

Reply:
`created_at` should not get current time it uses `auto_now_add` which adds new time only when actually row is created now modified. Are you sure that this is changing this time?

Reply:
https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.DateField.auto_now_add this is how `auto_now_add` works... if we dno't want this, we would need to get back to `default=datetime.datetime.now`
@cameel ? @rwrzesien

Reply:
It's fine. We don't want to change the `created_at` timestamp. The object is not being created after all.

We would want to change `modified_at` if there was one. But there isn't because the object is never supposed to be modified in normal circumstances.

Actually, I said in #604 that we don't need modification timestamps in models that are considered read-only like this one but I did not consider migrations. I assumed that anyone changing the state outside of normal code (e.g. with a command-line SQL client) would not update the timestamps anyway. But migrations are kinda in between those situations. Maybe they would have a use after all...

Reply:
Okey, my bad. `created_at` change for me, when I tested for the first time, because I make initial messages before that field was in the model :/ that's why when they were created for the first time it took `datetime.now`.